<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1217</width>
    <height>1002</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(120, 120, 120);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>450</height>
         </size>
        </property>
        <property name="title">
         <string>Hardware</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QTreeWidget" name="parameters_treeWidget">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(208, 208, 208);</string>
           </property>
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>50</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>Progress</string>
        </property>
        <widget class="QProgressBar" name="progressBar">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>281</width>
           <height>21</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>Save</string>
        </property>
        <widget class="QPushButton" name="folder_pushButton">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>20</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Folder</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="filename_lineEdit">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>281</width>
           <height>20</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(208, 208, 208);</string>
         </property>
         <property name="text">
          <string>filename</string>
         </property>
        </widget>
        <widget class="QPushButton" name="save_pushButton">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>20</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
        <widget class="QTextEdit" name="comments_textEdit">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>281</width>
           <height>301</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(208, 208, 208);</string>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu Sans'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;User:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;Sample:&lt;br /&gt;Power:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;Wavelength:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;Freq: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>91</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Comments</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Filename</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>10</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>55</height>
         </size>
        </property>
        <property name="title">
         <string>Quick Control</string>
        </property>
        <widget class="QPushButton" name="acquire_pushButton">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>20</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Acquire</string>
         </property>
        </widget>
        <widget class="QPushButton" name="run_pushButton">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>20</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Run</string>
         </property>
        </widget>
        <widget class="QPushButton" name="stop_pushButton">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>20</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
        <widget class="QPushButton" name="view_pushButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>View</string>
         </property>
        </widget>
        <widget class="QWidget" name="horizontalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>530</x>
           <y>10</y>
           <width>321</width>
           <height>41</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="set_power_layout"/>
        </widget>
        <widget class="QPushButton" name="show_beam_explorer_button">
         <property name="geometry">
          <rect>
           <x>330</x>
           <y>20</y>
           <width>171</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Show Beam Explorer</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>5</number>
        </property>
        <widget class="QWidget" name="spectro_tab">
         <attribute name="title">
          <string>Spectrum View</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="SLM_tab">
         <attribute name="title">
          <string>SLM View</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="spatial_tab">
         <attribute name="title">
          <string>Spatial Calibration</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="temporal_tab">
         <attribute name="title">
          <string>Temporal Calibration</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="parameter_tab">
         <attribute name="title">
          <string>Parameter View</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Utilities</string>
         </attribute>
         <widget class="QGroupBox" name="groupBox_8">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>851</width>
            <height>141</height>
           </rect>
          </property>
          <property name="title">
           <string>Background </string>
          </property>
          <widget class="QPushButton" name="Acquire_bg_pushButton">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>80</y>
             <width>221</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Acquire Background</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_9">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>30</y>
             <width>221</width>
             <height>41</height>
            </rect>
           </property>
           <property name="text">
            <string>Number of scans to average</string>
           </property>
          </widget>
          <widget class="QSpinBox" name="bg_scans_spinBox">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>40</y>
             <width>101</width>
             <height>22</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="suffix">
            <string> scan(s)</string>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
           <property name="value">
            <number>5</number>
           </property>
          </widget>
          <widget class="QLineEdit" name="bg_file_lineEdit">
           <property name="geometry">
            <rect>
             <x>360</x>
             <y>40</y>
             <width>211</width>
             <height>25</height>
            </rect>
           </property>
           <property name="text">
            <string>Background_file</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="bg_checkBox">
           <property name="geometry">
            <rect>
             <x>500</x>
             <y>80</y>
             <width>161</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Correct Background</string>
           </property>
          </widget>
          <widget class="QPushButton" name="select_bg_pushButton">
           <property name="geometry">
            <rect>
             <x>260</x>
             <y>80</y>
             <width>211</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Select Background</string>
           </property>
          </widget>
         </widget>
         <widget class="QGroupBox" name="groupBox_6">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>160</y>
            <width>851</width>
            <height>121</height>
           </rect>
          </property>
          <property name="title">
           <string>Time resolved Measurement</string>
          </property>
          <widget class="QLineEdit" name="kinetic_lineEdit">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>40</y>
             <width>261</width>
             <height>20</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>1:5:5 p6:10:16</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_19">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>20</y>
             <width>111</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Enter time intervall in s</string>
           </property>
          </widget>
          <widget class="QPushButton" name="kinetic_run_pushButton">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>70</y>
             <width>181</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Run Time resolved Measurement</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_20">
           <property name="geometry">
            <rect>
             <x>570</x>
             <y>30</y>
             <width>251</width>
             <height>71</height>
            </rect>
           </property>
           <property name="text">
            <string>Enter in format: 
Start:Stepnumber:Stop  Start:Stepnum... 
 &quot;o&quot; for open shutter    &quot;c&quot; for close shutter 
 &quot;p&quot; prefix for probe measurement</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_21">
           <property name="geometry">
            <rect>
             <x>290</x>
             <y>30</y>
             <width>271</width>
             <height>71</height>
            </rect>
           </property>
           <property name="text">
            <string>This measurement acquires spectra according  
 to a user-defined pattern. </string>
           </property>
          </widget>
         </widget>
         <widget class="QLabel" name="label_vertical_calibration_box">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>290</y>
            <width>231</width>
            <height>19</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="text">
           <string>  Phase to Greyscale - LUT File Measurement</string>
          </property>
         </widget>
         <widget class="QGroupBox" name="LUT_calibration_groupbox">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>300</y>
            <width>851</width>
            <height>281</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
            <underline>false</underline>
            <kerning>false</kerning>
           </font>
          </property>
          <property name="title">
           <string/>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <widget class="QLabel" name="LUT_calib_label_int_time">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>131</width>
             <height>19</height>
            </rect>
           </property>
           <property name="text">
            <string>Integration Time (ms)</string>
           </property>
          </widget>
          <widget class="PlotWidget" name="LUT_calib_plot_layout" native="true">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>280</x>
             <y>10</y>
             <width>551</width>
             <height>251</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="measure_LUT_calib">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>150</y>
             <width>231</width>
             <height>27</height>
            </rect>
           </property>
           <property name="text">
            <string>Measure Spectrum - Phase2Grey Calibration</string>
           </property>
          </widget>
          <widget class="QDoubleSpinBox" name="LUT_int_time_doubleSpinBox">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>30</y>
             <width>91</width>
             <height>21</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="suffix">
            <string> ms</string>
           </property>
           <property name="maximum">
            <double>10000.000000000000000</double>
           </property>
           <property name="value">
            <double>10.000000000000000</double>
           </property>
          </widget>
          <widget class="QLabel" name="LUT_calib_label_num_spectra">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>70</y>
             <width>111</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>Number of Spectra</string>
           </property>
          </widget>
          <widget class="QSpinBox" name="LUT_calib_spectra_avg_spinBox">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>70</y>
             <width>101</width>
             <height>22</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="suffix">
            <string>  spectra</string>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
           <property name="value">
            <number>5</number>
           </property>
          </widget>
          <widget class="QLabel" name="LUT_calib_scans_number_spectra">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>110</y>
             <width>111</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>Number of Scans</string>
           </property>
          </widget>
          <widget class="QSpinBox" name="LUT_calib_scans_number_spinBox">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>110</y>
             <width>101</width>
             <height>22</height>
            </rect>
           </property>
           <property name="toolTipDuration">
            <number>-2</number>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="suffix">
            <string>  scans</string>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
           <property name="value">
            <number>2</number>
           </property>
          </widget>
          <widget class="QLineEdit" name="LUT_Data_file_lineEdit">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>190</y>
             <width>101</width>
             <height>25</height>
            </rect>
           </property>
           <property name="text">
            <string>Data_File</string>
           </property>
          </widget>
          <widget class="QPushButton" name="generate_LUT_calib">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>230</y>
             <width>231</width>
             <height>27</height>
            </rect>
           </property>
           <property name="text">
            <string>Generate LUT File - Phase2Grey Calibration</string>
           </property>
          </widget>
          <widget class="QPushButton" name="select_LUT_Data_file_pushButton">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>190</y>
             <width>111</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Select Data File</string>
           </property>
          </widget>
         </widget>
         <zorder>groupBox_8</zorder>
         <zorder>groupBox_6</zorder>
         <zorder>LUT_calibration_groupbox</zorder>
         <zorder>label_vertical_calibration_box</zorder>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1217</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTransmission">
    <property name="title">
     <string>Transmission</string>
    </property>
    <widget class="QMenu" name="menuDisplay_mode">
     <property name="title">
      <string>Display mode</string>
     </property>
     <addaction name="actionNo_correction"/>
     <addaction name="actionTransmission"/>
     <addaction name="actionAbsorbance"/>
     <addaction name="actionAbsorption"/>
    </widget>
    <addaction name="actionChoose_reference_spectrum"/>
    <addaction name="separator"/>
    <addaction name="menuDisplay_mode"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTransmission"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionMeasure_absorption">
   <property name="text">
    <string>Measure absorption</string>
   </property>
  </action>
  <action name="actionMeasure_Absorptance">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Measure Absorptance</string>
   </property>
  </action>
  <action name="actionTransmission">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Transmission</string>
   </property>
  </action>
  <action name="actionAbsorption">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Absorption</string>
   </property>
  </action>
  <action name="actionAbsorbance">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Absorbance</string>
   </property>
  </action>
  <action name="actionNo_correction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>No correction</string>
   </property>
  </action>
  <action name="actionSelected_reference_spectrum">
   <property name="text">
    <string>Selected reference spectrum</string>
   </property>
  </action>
  <action name="actionChoose_reference_spectrum">
   <property name="text">
    <string>Choose reference spectrum</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
